packages/block-library/src/query-no-results/index.php

```diff
@@ -10,6 +10,8 @@
  *
  * @since 6.0.0
  *
+ * @global WP_Query $wp_query              WordPress Query object.
```

Comment:
```suggestion
 * @global WP_Query $wp_query WordPress Query object.
```

If there are multiple global annotations, you need to insert a space according to the longest annotation. However, here there is only one global annotation, so a single space would be fine.